setup.py

```diff
@@ -3,7 +3,7 @@
     long_description = fh.read()
 
 setuptools.setup(
-    name='tell_me_your_secrets',
+    name='tell_me_your_secrets_tmp',
```

Comment:
Module Name Change Required.

Reply:
Oh sorry - this was me debugging something when using the package in a project :)